Dockerfile

```diff
@@ -25,9 +25,10 @@ RUN apt-get update && \
     build-essential \
     m4 \
     git-core \
-    python-sphinx \
+    python3-sphinx \
     mysql-client \
     nodejs \
+    npm \
```

Comment:
what is npm used for again?

Reply:
It's used on line 43 to install node-gyp.